src/Modules/Trinity.DynamicCluster/Trinity.ServiceFabric.GraphEngineService/GraphEngineService.cs

```diff
@@ -84,7 +84,7 @@ internal TrinityErrorCode Start()
                 {
                     TrinityServer = AssemblyUtility.GetAllClassInstances(
                         t => t != typeof(TrinityServer) ?
-                        t.GetConstructor(new Type[] { }).Invoke(new object[] { }) as TrinityServer :
+                        t.GetConstructor(new Type[] { })?.Invoke(new object[] { }) as TrinityServer :
```

Comment:
TODO we should insert .Where(_ => _!=null) here to prevent a bad class to terminate our search

Reply:
@yatli Cool - I'll drop the null check